libs/src/api/entityManager.ts

```diff
@@ -21,17 +21,23 @@ export interface PlaylistOperationResponse {
    * Block number of playlist transaction
    */
   blockNumber: Nullable<number>
-  /**
-   * ID of playlist being modified
-   */
-  playlistId: Nullable<number>
   /**
    * String error message returned
    */
   error: Nullable<string>
 }
-
-const { encodeHashId, decodeHashId } = Utils
+type PlaylistTrack = { time: number; metadata_time?: number; track: number }
+
+type PlaylistParam = {
+  playlist_id: number
+  playlist_name: string
+  artwork?: { file?: File; url?: string }
+  playlist_contents: { track_ids: PlaylistTrack[] | number[] }
```

Comment:
what's the reason for allowing track ids to be an array or numbers or array of playlistTrack?

Reply:
yea `number[]` is for createPlaylist when you upload a playlist. the trackIds are just numbers in that case. i added a comment to clarify this but maybe we should just add the block height timestamps here. 

```
https://github.com/AudiusProject/audius-client/blob/e20f34a0ff3a755cf8732ecf54f16552911ee4d7/packages/web/src/common/services/audius-backend/AudiusBackend.ts#L1513
```

Reply:
yea i just moved the timestamp generation to the client